packages/roosterjs-editor-plugins/lib/HyperLink/HyperLink.ts

```diff
@@ -33,13 +28,30 @@ export default class HyperLink implements EditorPlugin {
      */
     public initialize(editor: Editor): void {
         this.editor = editor;
+        this.disposers = this.getTooltipCallback
+            ? [
+                  editor.addDomEventHandler('mouseover', this.onMouse),
+                  editor.addDomEventHandler('mouseout', this.onMouse),
+              ]
+            : [];
     }
 
+    private onMouse = (e: MouseEvent) => {
+        let href = this.tryGetHref(e.srcElement);
+        if (href) {
```

Comment:
When there's no href (e.g. you're mouse over text element), should we remove the title?

Reply:
It doesn't matter where the mouse is moving to, but where it is moving from, so once it is moved out of an A tag, we clear the title.

Reply:
ah, mouseout--but how about when you use your keyboard switch tab or minimize the browser, then move the keyboard elsewhere? Is it guaranteed that mouseout on anchor will always occur?

Reply:
Tried, yes, it will always be triggered.

Reply:
yes, it will be triggered.